include/blosc2.h

```diff
@@ -1804,6 +1804,34 @@ BLOSC_EXPORT int blosc2_schunk_get_chunk(blosc2_schunk *schunk, int64_t nchunk,
 BLOSC_EXPORT int blosc2_schunk_get_lazychunk(blosc2_schunk *schunk, int64_t nchunk, uint8_t **chunk,
                                              bool *needs_free);
 
+/**
+ * @brief Fill buffer with schunk slice.
+ *
+ * @param schunk The super-chunk from where to extract a slice.
+ * @param start The coordinates where the slice will begin (0 indexed).
+ * @param stop The coordinates where the slice will end.
```

Comment:
'Coordinates' is more for multidimensionality.  Suggestions:

"The coordinates where the slice will begin (0 indexed)." -> Index (0-based) where the slice begins.
"The coordinates where the slice will end." -> The first index (0-based) that is not in the selected slice.

Reply:
Fixed in 1208e4e76f0f085baf1a4e02a8cc276654d40b0f.